BrowserKit/Package.swift

```diff
@@ -5,7 +5,8 @@ import PackageDescription
 let package = Package(
     name: "BrowserKit",
     platforms: [
-        .iOS(.v15)
+        .iOS(.v15),
+        .macOS(.v10_14)
```

Comment:
Needed to define the platform for macOS apparently to be able to run `swift run` from the command line for this Package